pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+				status_history: statusHistory,
+				tally,
+				timeline: proposalTimeline,
+				title: subsquareTitle || 'Untitled',
+				topic: topicFromType,
+				track_no: !isNaN(trackNumber) ? trackNumber : null,
+				type: type,
+				user_id: 1
+			};
+			await getContentSummary(post, network, true);
+
+			delete post?.content;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)